tests/canopen/test_canopen_examples.py

```diff
@@ -22,43 +22,43 @@ def arguments(setup_descriptor):
 def test_connection_example(arguments, script_runner):
     script_path = "examples/canopen/can_connection.py"
     result = script_runner.run([script_path, *arguments])
-    assert result.returncode == 0
+    assert result.returncode == 0, f"Script failed with stderr:\n{result.stderr}"
```

Comment:
I had some failures in jenkins and all you got is the error code. Now it is possible to debug